src/DeviceInterface/InterfacePrototype.js

```diff
@@ -453,7 +455,8 @@ class InterfacePrototype {
         dataPromise.then(response => {
             if (response) {
                 const data = response.success || response
-                if (config.type === 'identities') {
+                if (mainType === 'identities') {
+                    // TODO: add trigger field type (subtype)
```

Comment:
This is what's left to do here. If you can't make it before Sam's release, we can follow up next week.